spec/controllers/bales_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe BalesController, type: :controller do
+  let(:json_response) { JSON.parse(response.body, symbolize_keys: true) }
+
+  describe 'POST#create' do
+    let!(:bale) { FactoryBot.create(:bale) }
+    let(:b_serializer) { BaleSerializer }
+
+    context 'Success case' do
+      it 'should return success' do
+        post :create, params: { bale: { weight: bale.weight, material: bale.material } }
```

Comment:
Add a method to make this call, something like ` create_bale_call(weight, material) ` and then use it when you need to